src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentServiceSpec.scala

```diff
@@ -586,36 +586,6 @@ class ConsignmentServiceSpec extends AnyFlatSpec with MockitoSugar with ResetMoc
     exception.getMessage should equal("Existing consignment upload status is 'InProgress', so cannot start new upload")
   }
 
-  "addSeriesStatus" should "update the consignment status for a series" in {
```

Comment:
The `"updateSeriesIdOfConsignment" should "update the seriesId for a given consignment"` should be updated to test whether the status gets updated in the same call.